scripts/gluster-ansible-infra/infra_firewall_config.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# setting up virtual environment
+virtualenv --system-site-packaes env
+source env
+
+# install dependency packages
+yum install libselinux-python
+yum install gcc python-virtualenv
+pip install ansible molecule docker-py
+
+# start and enable Docker service
+systemctl start docker
+systemctl enable docker
+
+# Ensure that your user is in the docker group.
+sudo groupadd docker
+sudo usermod -aG docker $USER
+
+# Verify everything works with the following:
+docker run hello-world
+
+# if selinux is enabled
+selinuxenabled
+if [ $? -ne 0 ]
+then
+    sudo yum install libselinux-python
+
+# basic dependencies for the tests
```

Comment:
This is the only section I want you to keep in this script. Move everything else into a new folder on gluster-ansible-infra that says `tests/run-centos-ci.sh` and then call that script from this one right after you clone it.